zebra-rpc/src/rpc.rs

```diff
@@ -0,0 +1,35 @@
+//! Zebra supported RPC methods.
```

Comment:
Code Structure Question: Can we call this module something like `zebra_rpc::methods`.

Another option is `server_methods`. What do you think?

Reply:
https://github.com/ZcashFoundation/zebra/pull/3589/commits/c16b3bec513ec844ca62551cbd3dd53a14014b13